steps/dotnetcore/1.0.0/step.yml

```diff
@@ -0,0 +1,37 @@
+title: .NET Core
```

Comment:
Reason I made it that, was because in the workflow editor you only see a small part of the text. When adding "Install" in front of it, you loose context in the Workflow editor. So maybe "Add .NET Core" ?